job/server/src/test/java/alluxio/job/command/CommandHandlingExecutorTest.java

```diff
@@ -94,8 +87,8 @@ public void heartbeat() throws Exception {
         .thenReturn(Lists.newArrayList(command.build()));
 
     mCommandHandlingExecutor.heartbeat();
-    ExecutorService executorService =
-        Whitebox.getInternalState(mCommandHandlingExecutor, "mCommandHandlingService");
+    ExecutorService executorService = AlluxioMockUtil.getInternalState(mCommandHandlingExecutor,
+        mCommandHandlingExecutor.getClass().getDeclaredField("mCommandHandlingService"));
```

Comment:
I see this pattern is being repeated a few times:

```java
getInternalState(obj, obj.getClass().getDeclaredField("fieldName"))
```

Can you add an overloaded method for `getInternalState(Object, String)` that takes the field name as a string?

Reply:
done~